README.md

```diff
@@ -242,9 +242,12 @@ For works that have used MTEB for benchmarking, you can find them on the [leader
 | [BUCC](https://comparable.limsi.fr/bucc2018/bucc2018-task.html)                                                                                                       | [mteb/bucc-bitext-mining](https://huggingface.co/datasets/mteb/bucc-bitext-mining)                                                   | BUCC bitext mining dataset                                                                                                                                                                                       | BitextMining       | s2s      |          4 |              0 |            0 |        641684 |                  0 |                0 |             101.3 |
 | [Tatoeba](https://github.com/facebookresearch/LASER/tree/main/data/tatoeba/v1)                                                                                        | [mteb/tatoeba-bitext-mining](https://huggingface.co/datasets/mteb/tatoeba-bitext-mining)                                             | 1,000 English-aligned sentence pairs for each language based on the Tatoeba corpus                                                                                                                               | BitextMining       | s2s      |        112 |              0 |            0 |          2000 |                  0 |                0 |              39.4 |
 | [Bornholm parallel](https://aclanthology.org/W19-6138/)                                                                                                               | [strombergnlp/bornholmsk_parallel](https://huggingface.co/datasets/strombergnlp/bornholmsk_parallel)                                 | Danish Bornholmsk Parallel Corpus.                                                                                                                                                                               | BitextMining       | s2s      |          2 |            100 |          100 |           100 |               64.6 |             86.2 |              89.7 |
+| [DiaBLaBitextMining](https://inria.hal.science/hal-03021633) | [rbawden/DiaBLa](https://huggingface.co/datasets/rbawden/DiaBLa) | English-French Parallel Corpus. DiaBLa is an English-French dataset for the evaluation of Machine Translation (MT) for informal, written bilingual dialogue. | BitextMining | s2s | 1 | 5748 | 0 | 0 | 0 | 0 | 0 |
+| [FloresBitextMining](https://huggingface.co/datasets/facebook/flores) | [facebook/flores](https://huggingface.co/datasets/facebook/flores) | FLORES is a benchmark dataset for machine translation between English and low-resource languages. | BitextMining | s2s | 200 | 997 | 0 | 0 | 0 | 0 | 0 |
```

Comment:
Can we add the test split num samples now that it's added? I think it's also 997 but not 100% sure

Reply:
It's 1012 samples for the `devtest` slpit

Reply:
```suggestion
| [FloresBitextMining](https://huggingface.co/datasets/facebook/flores) | [facebook/flores](https://huggingface.co/datasets/facebook/flores) | FLORES is a benchmark dataset for machine translation between English and low-resource languages. | BitextMining | s2s | 200 | 0 | 997 | 1012 | 0 | 0 | 0 |
```